assess.dom.js

```diff
@@ -519,6 +519,25 @@ QUnit.test("duplication",
             "The two paragraphs should have the same text."
         );
 
+        assert.notEqual(
+            paras[0].id,
+            paras[1].id,
```

Comment:
try something like `assert.ok((paras[0].id == null && paras[1].id == null) || paras[0].id != paras[1].id, ...`

this would be more robust if in the future we change the selection code to not use an ID, and then the element to be duplicated wouldn't even have an ID.

Reply:
Resolved in https://github.com/portsoc/ws_dom/pull/4/commits/6e8a53d851184c296e271eb8b8aeb76a0eea85e5